### PR TITLE
[NUI] Binds GetCurrentProperty to Animatable class

### PR DESCRIPTION
**Pull request by bshsqa**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/Animation/Animatable.cs [hunk 1]
```diff
@@ -199,6 +199,18 @@ public PropertyValue GetProperty(int index)
             return ret;
         }
 
+        /// <summary>
+        /// Retrieves a property value.
```
Comment (hinohie) [inline src/Tizen.NUI/src/public/Animation/Animatable.cs:203]:
```suggestion
        /// Retrieves the latest rendered frame value of the property.
```

I think we'd better match the comment with dali-core side comments. Actually, should be difference with `GetProperty` API comments.

https://github.com/dalihub/dali-core/blob/master/dali/public-api/object/handle.h#L480

**Review by hinohie — COMMENTED** [review]
Change the `CaptureTest.cs` is what you intented?

File: src/Tizen.NUI/src/public/Animation/Animatable.cs (hunk 1 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/public/Animation/Animatable.cs]: Oh. good good. I fixed.

**Review by hinohie — APPROVED** [review]
Looks good to me

**Pull request merged**